buddyboss/buddyboss-platform
# PROD-652 - Create album button issue for group - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3261  .

### How to test the changes in this Pull Request:

1. Make sure "Enable albums in groups" option is enabled from BuddyBoss > Settings > Media
2. In BuddyBoss > Settings > Media > Photos > Untick Enable album in Profiles option.
3. See the Create Album button is showing in Group

### Proof Screenshots or Video

https://share.vidyard.com/watch/TFedB71eHnp64dwmsBfyhJ?

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

**Jira issue** : [PROD-652]

[PROD-652]: https://buddyboss.atlassian.net/browse/PROD-652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
